src/locales/en/translation.json

```diff
@@ -428,8 +428,13 @@
     "settings": {
       "cancel": "Cancel",
       "contacts": "Contacts",
-      "exportPrivateKey": "Export Private Key",
-      "lock": "Lock",
+      "exportPrivateKey": {
+        "confirm": "I understand, reveal my private key",
+        "copied": "Private key copied.",
+        "hint1": "The private key consists of a string of characters, and owning the private key is equivalent to owning the asset ownership.",
```

Comment:
"owning the asset ownership"
we need to improve this

Reply:
@donouwens, Can you please provide me an updated copy for this section? Current one was copied from an old extension.

Reply:
I like 1.